shared/message/platform/NUSenseData.proto

```diff
@@ -34,6 +34,18 @@ package message.platform;
 // Message mainly for NUSense <-> NUC communication. NUSense will encode this message via nanopb after it queries all
 // servo states and send it to the NUC.
 message Servo {
+
+    message PacketCounts {
+        /// The number of successes.
+        uint32 total = 1;
+        /// The number of timeouts.
+        uint32 timeouts = 2;
+        /// The number of CRC-errors.
+        uint32 crc_errors = 3;
+        /// The number of packet-errors.
+        uint32 errors = 4;
```

Comment:
What's the difference between a crc error and a packet error?

```suggestion
        /// The number of CRC-errors.
        uint32 crc_errors = 3;
        /// The number of packet-errors.
        uint32 packet_errors = 4;
```

Reply:
A CRC error is when the CRC of the received packet does not match the computed CRC, i.e. the received packet has been corrupted.

A packet-error is when the received packet has an error-code in its error-field which could mean a few things depending on the context, out-of-range data-length, inaccessible address, attempting to write to a read-only register, a mismatched CRC on the servo's side, etc.